### DISCUSSION TITLE
Exporting to raster

### DISCUSSION THREAD

**Discussion by CavemanEx** [fatiando/community#94] [General]
Hi team,
I'm a huge fan of your work and have had a ton of fun gridding everything from ground geophysics to soil samples.

One issue I had when first starting out is exporting the final xarrays to georeferenced rasters. It's probably a no brainer for most people on here, but this was my first time working with xarrays.

I followed a tutorial by opensourceoptions on youtube about working with xarrays and netCDF but ended up having troubles with the coordinate systems.

In the end, I ended up using this short code at the end of the chain to export as a tif with the correct coordinate system:

crs = CRS.from_epsg(XXXXX)
grid.rio.set_spatial_dims("easting", "northing", inplace=True)
grid.rio.write_crs(input_crs=crs, inplace=True)
fn = (Save location)/File_Name.tif
grid.rio.to_raster(fn)

It's probably not the cleanest way but you might consider adding a short section on exporting the final grids to the tutorial section of the gridder.

Thanks a ton for this awesome project,
Harley

**Comment by LL-Geo** [comment]
Hi Harley,

Thanks for posting this! 😄 

Yea, it will take some time to get used to Xarray. It handles netCDF quite easily, but [rioxarray](https://corteva.github.io/rioxarray) and [Rasterio](https://rasterio.readthedocs.io/) are tools to deal with the raster data. 

There are some nice examples in [Fatiando-data](https://github.com/fatiando-data), which shows how to get raw data into beautiful data sets. For example in [here](https://github.com/fatiando-data/lightning-creek-magnetic-grid/blob/main/prepare.ipynb) in [12], you could set up the coordinate information or other metadata and proper field use `grid.attrs = {
    "crs": str(projection), ...
}` , then you could either save it to netcdf `grid.to_netcdf` or geotiff by `grid.rio.to_raster()`.

Thanks for the suggestion, I think we should put it somewhere or link it along with the tutorial section. So people will be easy to find it! 😄
